code/datums/diseases/advance/symptoms/choking.dm

```diff
@@ -50,4 +50,66 @@ Bonus
 /datum/symptom/choking/proc/Choke(mob/living/M, datum/disease/advance/A)
 	var/get_damage = sqrt(21+A.totalStageSpeed()*0.5)+sqrt(16+A.totalStealth()*5)
 	M.adjustOxyLoss(get_damage)
+	return 1
+
+/*
+//////////////////////////////////////
+
+Asphyxiation
+
+	Very very noticable.
+	Decreases stage speed.
+	Decreases transmittablity.
+
+Bonus
+	Inflicts large spikes of oxyloss
+	Introduces Asphyxiating drugs to the system
+	Causes cardiac arrest on dying victims.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/asphyxiation
+
+	name = "Acute respiratory distress syndrome"
+	stealth = -2
+	resistance = -0
+	stage_speed = -1
+	transmittable = -2
+	level = 7
+	severity = 3
+
+/datum/symptom/asphyxiation/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/M = A.affected_mob
+		switch(A.stage)
+			if(3, 4)
+				M << "<span class='warning'><b>[pick("Your windpipe feels thin.", "Your lungs feel small.")]</span>"
+				Asphyxiate_stage_3_4(M, A)
+				M.emote("gasp")
+			else
+				M << "<span class='userdanger'>[pick("Your lungs hurt!", "It hurts to breath!")]</span>"
+				Asphyxiate(M, A)
+				M.emote("gasp")
+				if(M.getOxyLoss() >= 120)

```

Comment:
o yeahhh super good reba;lance instead of stun and kill in one symptom it just kills u outright oh yeaaaaah